### PR TITLE
Println changed to Log4j-GabySFB

### PR DESCRIPTION
**Pull request by GabySFB**

### CODE REVIEW

File: GabrielaRamirez/GradleExample/workspace/src/main/java/challenge/Icecream.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
+package challenge;
+
+import org.apache.log4j.Logger;
+
+public class Icecream {
+
+    public static Logger logJava = Logger.getLogger(Icecream.class);
```
Comment (globant-bootcamp) [inline GabrielaRamirez/GradleExample/workspace/src/main/java/challenge/Icecream.java:7]: variables final need to be in uppercase

File: GabrielaRamirez/GradleExample/workspace/src/main/java/challenge/Main.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
+package challenge;
+
+import org.apache.log4j.Logger;
+
+public class Main {
+
+    public static Logger logJava = Logger.getLogger(Main.class);
```
Comment (globant-bootcamp) [inline GabrielaRamirez/GradleExample/workspace/src/main/java/challenge/Main.java:7]: variables final need to be in uppercase

File: GabrielaRamirez/GradleExample/workspace/src/main/java/challenge/Main.java [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+    public static void main(String[] args){
+        Icecream icecream = new Icecream();
+        logJava.info(icecream.BRAND);
+        icecream.printFlavour("coconut");
+        icecream.topping("chocolate");
+
+
```
Comment (globant-bootcamp) [inline GabrielaRamirez/GradleExample/workspace/src/main/java/challenge/Main.java:15]: delete unnecessary blanks

File: GabrielaRamirez/GradleExample/workspace/src/main/java/challenge/Main.java [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+        SnowIceCream nieve = new SnowIceCream();
+        nieve.printFlavour("Mamey");
+        nieve.topping("nuts", "chocolate");
+
+        Popsicle popsicle = new Popsicle();
+        popsicle.printFlavour("banaba");
```
Comment (globant-bootcamp) [inline GabrielaRamirez/GradleExample/workspace/src/main/java/challenge/Main.java:21]: put this hardcoded string inside static final variables

File: GabrielaRamirez/GradleExample/workspace/src/main/java/challenge/Main.java [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+        popsicle.topping("strawberry", "chocolate", "nuts");
```
Comment (globant-bootcamp) [inline GabrielaRamirez/GradleExample/workspace/src/main/java/challenge/Main.java:22]: put this hardcoded string inside static final variables

File: GabrielaRamirez/GradleExample/workspace/src/main/java/hackkerank/day1/IfElse.java [hunk 6]
```diff
@@ -0,0 +1,35 @@
+package hackkerank.day1;
+import challenge.Icecream;
+import org.apache.log4j.Logger;
+
+import java.util.Scanner;
+
+public class IfElse {
+
+    public static Logger logJava = Logger.getLogger(IfElse.class);
```
Comment (globant-bootcamp) [inline GabrielaRamirez/GradleExample/workspace/src/main/java/hackkerank/day1/IfElse.java:9]: final variables must be in uppercase

File: GabrielaRamirez/GradleExample/workspace/src/main/java/hackkerank/day1/IfElse.java [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+
+    private static final Scanner scanner = new Scanner(System.in);
```
Comment (globant-bootcamp) [inline GabrielaRamirez/GradleExample/workspace/src/main/java/hackkerank/day1/IfElse.java:11]: final variables must be in uppercase

File: GabrielaRamirez/GradleExample/workspace/src/main/java/hackkerank/day1/IfElse.java [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+
+    public static void main(String[] args) {
+        int N = scanner.nextInt();
```
Comment (globant-bootcamp) [inline GabrielaRamirez/GradleExample/workspace/src/main/java/hackkerank/day1/IfElse.java:14]: Improve variable name, lowerCamelCase

File: GabrielaRamirez/GradleExample/workspace/src/main/java/hackkerank/day1/IfElse.java [hunk 9]
```diff
@@ -0,0 +1,35 @@
...
+        scanner.skip("(\r\n|[\n\r\u2028\u2029\u0085])?");
+
+        scanner.close();
+
+        if (N%2==0 & 2<=N & N<=5){
```
Comment (globant-bootcamp) [inline GabrielaRamirez/GradleExample/workspace/src/main/java/hackkerank/day1/IfElse.java:19]: format the code,

File: GabrielaRamirez/GradleExample/workspace/src/main/java/hackkerank/day1/StdinI.java [hunk 10]
```diff
@@ -0,0 +1,20 @@
+package hackkerank.day1;
+import org.apache.log4j.Logger;
+
+import java.util.Scanner;
+
+public class StdinI {
+
+    public static Logger logJava = Logger.getLogger(StdinI.class);
```
Comment (globant-bootcamp) [inline GabrielaRamirez/GradleExample/workspace/src/main/java/hackkerank/day1/StdinI.java:8]: final variables must be in uppercase

File: GabrielaRamirez/GradleExample/workspace/src/main/java/hackkerank/day1/StdinI.java [hunk 11]
```diff
@@ -0,0 +1,20 @@
...
+
+    public static void main(String[] args) {
+        Scanner scan = new Scanner(System.in);
+        int a = scan.nextInt();
```
Comment (globant-bootcamp) [inline GabrielaRamirez/GradleExample/workspace/src/main/java/hackkerank/day1/StdinI.java:12]: Improve the variable name, lowerCamelCase

File: GabrielaRamirez/GradleExample/workspace/src/main/java/hackkerank/day1/StdinI.java [hunk 12]
```diff
@@ -0,0 +1,20 @@
...
+        int b = scan.nextInt();
```
Comment (globant-bootcamp) [inline GabrielaRamirez/GradleExample/workspace/src/main/java/hackkerank/day1/StdinI.java:13]: Same

File: GabrielaRamirez/GradleExample/workspace/src/main/java/hackkerank/day1/StdinI.java [hunk 13]
```diff
@@ -0,0 +1,20 @@
...
+        int c = scan.nextInt();
```
Comment (globant-bootcamp) [inline GabrielaRamirez/GradleExample/workspace/src/main/java/hackkerank/day1/StdinI.java:14]: Same

File: GabrielaRamirez/GradleExample/workspace/src/main/java/hackkerank/day1/StdinII.java [hunk 14]
```diff
@@ -0,0 +1,20 @@
...
+
+import java.util.Scanner;
+
+public class StdinII {
+
+    public static Logger logJava = Logger.getLogger(StdinII.class);
+
+    public static void main(String[] args) {
+        Scanner scan = new Scanner(System.in);
+        int a = scan.nextInt();
```
Comment (globant-bootcamp) [inline GabrielaRamirez/GradleExample/workspace/src/main/java/hackkerank/day1/StdinII.java:12]: Improve the variable name, lowerCamelCase

File: GabrielaRamirez/GradleExample/workspace/src/main/java/hackkerank/day1/StdinII.java [hunk 15]
```diff
@@ -0,0 +1,20 @@
...
+        int b = scan.nextInt();
```
Comment (globant-bootcamp) [inline GabrielaRamirez/GradleExample/workspace/src/main/java/hackkerank/day1/StdinII.java:13]: Same

File: GabrielaRamirez/GradleExample/workspace/src/main/java/hackkerank/day1/StdinII.java [hunk 16]
```diff
@@ -0,0 +1,20 @@
...
+        int c = scan.nextInt();
```
Comment (globant-bootcamp) [inline GabrielaRamirez/GradleExample/workspace/src/main/java/hackkerank/day1/StdinII.java:14]: Same

File: GabrielaRamirez/GradleExample/workspace/src/main/java/hackkerank/day2/Formatting.java [hunk 17]
```diff
@@ -0,0 +1,25 @@
+package hackkerank.day2;
+import org.apache.log4j.Logger;
+
+import java.util.Scanner;
+
+public class Formatting {
+
+    public static Logger logJava = Logger.getLogger(Formatting.class);
```
Comment (globant-bootcamp) [inline GabrielaRamirez/GradleExample/workspace/src/main/java/hackkerank/day2/Formatting.java:8]: final variables must be in uppercase

File: GabrielaRamirez/GradleExample/workspace/src/main/java/hackkerank/day2/Formatting.java [hunk 18]
```diff
@@ -0,0 +1,25 @@
...
+
+    public static final String SPACE = "================================";
+    public static final String STYLE = "%-15s%03d%n";
+
+    public static void main(String[] args) {
+        Scanner sc=new Scanner(System.in);
```
Comment (globant-bootcamp) [inline GabrielaRamirez/GradleExample/workspace/src/main/java/hackkerank/day2/Formatting.java:14]: Improve the variable name, lowerCamelCase

Comment (globant-bootcamp) [reply GabrielaRamirez/GradleExample/workspace/src/main/java/hackkerank/day2/Formatting.java:14]: format the code

File: GabrielaRamirez/GradleExample/workspace/src/main/java/hackkerank/day2/Formatting.java [hunk 19]
```diff
@@ -0,0 +1,25 @@
...
+        logJava.info(SPACE);
+        for(int i=0;i<3;i++)
+        {
+            String s1=sc.next();
```
Comment (globant-bootcamp) [inline GabrielaRamirez/GradleExample/workspace/src/main/java/hackkerank/day2/Formatting.java:18]: Improve the variable name, lowerCamelCase

File: GabrielaRamirez/GradleExample/workspace/src/main/java/hackkerank/day2/Formatting.java [hunk 20]
```diff
@@ -0,0 +1,25 @@
...
+            int x=sc.nextInt();
```
Comment (globant-bootcamp) [inline GabrielaRamirez/GradleExample/workspace/src/main/java/hackkerank/day2/Formatting.java:19]: Improve the variable name, lowerCamelCase

File: GabrielaRamirez/GradleExample/workspace/src/main/java/hackkerank/day2/inheritanceOne/Animal.java [hunk 21]
```diff
@@ -0,0 +1,14 @@
+package hackkerank.day2.inheritanceOne;
+
+import hackkerank.day1.IfElse;
+import org.apache.log4j.Logger;
+
+class Animal{
+
+    public static Logger logJava = Logger.getLogger(Animal.class);
```
Comment (globant-bootcamp) [inline GabrielaRamirez/GradleExample/workspace/src/main/java/hackkerank/day2/inheritanceOne/Animal.java:8]: final variables must be in uppercase

File: GabrielaRamirez/GradleExample/workspace/src/main/java/hackkerank/day2/inheritanceOne/Animal.java [hunk 22]
```diff
@@ -0,0 +1,14 @@
...
+
+    void walk()
+    {
+        logJava.info("I am walking");
```
Comment (globant-bootcamp) [inline GabrielaRamirez/GradleExample/workspace/src/main/java/hackkerank/day2/inheritanceOne/Animal.java:12]: put this hardcoded string inside static final variables

File: GabrielaRamirez/GradleExample/workspace/src/main/java/hackkerank/day2/inheritanceOne/Bird.java [hunk 23]
```diff
@@ -0,0 +1,16 @@
+package hackkerank.day2.inheritanceOne;
+
+import org.apache.log4j.Logger;
+
+class Bird extends Animal {
+
+    public static Logger logJava = Logger.getLogger(Bird.class);
```
Comment (globant-bootcamp) [inline GabrielaRamirez/GradleExample/workspace/src/main/java/hackkerank/day2/inheritanceOne/Bird.java:7]: final variables must be in uppercase

File: GabrielaRamirez/GradleExample/workspace/src/main/java/hackkerank/day2/inheritanceOne/Bird.java [hunk 24]
```diff
@@ -0,0 +1,16 @@
...
+
+    void fly() {
+        logJava.info("I am flying");
```
Comment (globant-bootcamp) [inline GabrielaRamirez/GradleExample/workspace/src/main/java/hackkerank/day2/inheritanceOne/Bird.java:10]: put this hardcoded string inside static final variables

File: GabrielaRamirez/GradleExample/workspace/src/main/java/hackkerank/day2/inheritanceOne/Bird.java [hunk 25]
```diff
@@ -0,0 +1,16 @@
...
+    }
+
+    void sing() {
+        logJava.info("I am singing");
```
Comment (globant-bootcamp) [inline GabrielaRamirez/GradleExample/workspace/src/main/java/hackkerank/day2/inheritanceOne/Bird.java:14]: put this hardcoded string inside static final variables

File: GabrielaRamirez/GradleExample/workspace/src/main/java/hackkerank/day2/inheritanceTwo/Arithmetic.java [hunk 26]
```diff
@@ -0,0 +1,10 @@
+package hackkerank.day2.inheritanceTwo;
+
+class Arithmetic{
+
+    int add(int x, int y){
```
Comment (globant-bootcamp) [inline GabrielaRamirez/GradleExample/workspace/src/main/java/hackkerank/day2/inheritanceTwo/Arithmetic.java:5]: Improve the variable name, lowerCamelCase

File: GabrielaRamirez/GradleExample/workspace/src/main/java/hackkerank/day2/inheritanceTwo/Arithmetic.java [hunk 27]
```diff
@@ -0,0 +1,10 @@
...
+        int c = x + y;
```
Comment (globant-bootcamp) [inline GabrielaRamirez/GradleExample/workspace/src/main/java/hackkerank/day2/inheritanceTwo/Arithmetic.java:6]: Improve the variable name, lowerCamelCase

File: GabrielaRamirez/GradleExample/workspace/src/main/java/hackkerank/day2/inheritanceTwo/InheritanceII.java [hunk 28]
```diff
@@ -0,0 +1,17 @@
+package hackkerank.day2.inheritanceTwo;
+
+import hackkerank.day1.IfElse;
+import org.apache.log4j.Logger;
+
+public class InheritanceII{
+
+    public static Logger logJava = Logger.getLogger(InheritanceII.class);
```
Comment (globant-bootcamp) [inline GabrielaRamirez/GradleExample/workspace/src/main/java/hackkerank/day2/inheritanceTwo/InheritanceII.java:8]: final variables must be in uppercase

File: GabrielaRamirez/GradleExample/workspace/src/main/java/hackkerank/day2/inheritanceTwo/InheritanceII.java [hunk 29]
```diff
@@ -0,0 +1,17 @@
...
+
+        public static void main(String []args){
+
+            Adder a = new Adder();
```
Comment (globant-bootcamp) [inline GabrielaRamirez/GradleExample/workspace/src/main/java/hackkerank/day2/inheritanceTwo/InheritanceII.java:12]: Improve the variable name, lowerCamelCase

File: GabrielaRamirez/GradleExample/workspace/src/main/java/hackkerank/day3/JavaArrayList.java [hunk 30]
```diff
@@ -0,0 +1,40 @@
...
+import org.apache.log4j.Logger;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+
+public class JavaArrayList {
+
+    public static Logger logJava = Logger.getLogger(IfElse.class);
```
Comment (globant-bootcamp) [inline GabrielaRamirez/GradleExample/workspace/src/main/java/hackkerank/day3/JavaArrayList.java:13]: final variables must be in uppercase

File: GabrielaRamirez/GradleExample/workspace/src/main/java/hackkerank/day3/JavaArrayList.java [hunk 31]
```diff
@@ -0,0 +1,40 @@
...
+
+    public static void main(String[] args) {
+
+        Scanner in = new Scanner(System.in);
+        int lines = in.nextInt();
+
+        List[] set = new ArrayList[lines];
```
Comment (globant-bootcamp) [inline GabrielaRamirez/GradleExample/workspace/src/main/java/hackkerank/day3/JavaArrayList.java:20]: It is recommended that the name of the collections and maps estart with singular name and them the name of the interface, example: numberList

File: GabrielaRamirez/GradleExample/workspace/src/main/java/hackkerank/day3/JavaArrayList.java [hunk 32]
```diff
@@ -0,0 +1,40 @@
...
+            }
+        }
+        int queryNumber = in.nextInt();
+        for (int i = 0; i < queryNumber; i++) {
+            int position = in.nextInt();
+            int line = in.nextInt();
+            try {
+                System.out.println(set[position - 1].get(line - 1));
+            } catch (IndexOutOfBoundsException e) {
+                System.out.println("ERROR!");
```
Comment (globant-bootcamp) [inline GabrielaRamirez/GradleExample/workspace/src/main/java/hackkerank/day3/JavaArrayList.java:35]: put this hardcoded string inside static final variables

File: GabrielaRamirez/GradleExample/workspace/src/main/java/hackkerank/day3/JavaHashset.java [hunk 33]
```diff
@@ -0,0 +1,28 @@
...
+
+    public static void main(String[] args) {
+
+        Scanner scan = new Scanner(System.in);
+        int size = scan.nextInt();
+        String[] pair_left = new String[size];
+        String[] pair_right = new String[size];
+
+        for (int i = 0; i < size; i++) {
+            pair_left[i] = scan.next();
```
Comment (globant-bootcamp) [inline GabrielaRamirez/GradleExample/workspace/src/main/java/hackkerank/day3/JavaHashset.java:17]: Improve the variable name, lowerCamelCase

File: GabrielaRamirez/GradleExample/workspace/src/main/java/hackkerank/day3/JavaHashset.java [hunk 34]
```diff
@@ -0,0 +1,28 @@
...
+            pair_right[i] = scan.next();
```
Comment (globant-bootcamp) [inline GabrielaRamirez/GradleExample/workspace/src/main/java/hackkerank/day3/JavaHashset.java:18]: Improve the variable name, lowerCamelCase

File: GabrielaRamirez/GradleExample/workspace/src/main/java/hackkerank/day3/JavaHashset.java [hunk 35]
```diff
@@ -0,0 +1,28 @@
...
+        }
+
+        Set<String> pairs = new HashSet<String>(size);
```
Comment (globant-bootcamp) [inline GabrielaRamirez/GradleExample/workspace/src/main/java/hackkerank/day3/JavaHashset.java:21]: It is recommended that the name of the collections and maps estart with singular name and them the name of the interface, example: numberList

File: GabrielaRamirez/GradleExample/workspace/src/main/java/hackkerank/day3/JavaMap.java [hunk 36]
```diff
@@ -0,0 +1,33 @@
...
+import java.util.Scanner;
+
+public class JavaMap {
+
+    public static void main(String[] argh) {
+
+        Scanner in = new Scanner(System.in);
+        int numberOfEntries = in.nextInt();
+        in.nextLine();
+        Map<String, Integer> map = new HashMap<>(numberOfEntries);
```
Comment (globant-bootcamp) [inline GabrielaRamirez/GradleExample/workspace/src/main/java/hackkerank/day3/JavaMap.java:14]: It is recommended that the name of the collections and maps estart with singular name and them the name of the interface, example: numberList

Comment (globant-bootcamp) [reply GabrielaRamirez/GradleExample/workspace/src/main/java/hackkerank/day3/JavaMap.java:14]: Improve the variable name

File: GabrielaRamirez/GradleExample/workspace/src/main/java/hackkerank/day3/ransomNote/MagazineNoteCheck.java [hunk 37]
```diff
@@ -0,0 +1,40 @@
...
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class MagazineNoteCheck {
+
+    public static void checkMagazine(List<String> magazine, List<String> note) {
+
+        Map<String, Integer> magazineWords = new HashMap<>();
```
Comment (globant-bootcamp) [inline GabrielaRamirez/GradleExample/workspace/src/main/java/hackkerank/day3/ransomNote/MagazineNoteCheck.java:11]: It is recommended that the name of the collections and maps estart with singular name and them the name of the interface, example: numberList

File: GabrielaRamirez/GradleExample/workspace/src/main/java/hackkerank/day3/ransomNote/MagazineNoteCheck.java [hunk 38]
```diff
@@ -0,0 +1,40 @@
...
+            if (magazineWords.containsKey(word)) {
+                int count = magazineWords.get(word);
+                if (count <= 0) {
+                    System.out.println("No");
+                    return;
+                }
+                count--;
+                magazineWords.put(word, count);
+            } else if (!magazineWords.containsKey(word)) {
+                System.out.println("No");
```
Comment (globant-bootcamp) [inline GabrielaRamirez/GradleExample/workspace/src/main/java/hackkerank/day3/ransomNote/MagazineNoteCheck.java:32]: put this hardcoded string inside static final variables

File: GabrielaRamirez/GradleExample/workspace/src/main/java/hackkerank/day3/ransomNote/MagazineNoteCheck.java [hunk 39]
```diff
@@ -0,0 +1,40 @@
...
+                return;
+            }
+        }
+
+        System.out.println("Yes");
```
Comment (globant-bootcamp) [inline GabrielaRamirez/GradleExample/workspace/src/main/java/hackkerank/day3/ransomNote/MagazineNoteCheck.java:37]: put this hardcoded string inside static final variables

File: GabrielaRamirez/GradleExample/workspace/src/main/java/hackkerank/day3/ransomNote/Main.java [hunk 40]
```diff
@@ -0,0 +1,30 @@
...
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.List;
+import java.util.stream.Stream;
+
+public class Main {
+
+    public static void main(String[] args) throws IOException {
+
+//        BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(System.in));
```
Comment (globant-bootcamp) [inline GabrielaRamirez/GradleExample/workspace/src/main/java/hackkerank/day3/ransomNote/Main.java:13]: Why this code was commented?

**Review by globant-bootcamp — CHANGES_REQUESTED** [review]

File: GabrielaRamirez/GradleExample/workspace/src/main/java/hackkerank/day3/JavaMap.java (hunk 36 above)
Comment (GabySFB) [reply GabrielaRamirez/GradleExample/workspace/src/main/java/hackkerank/day3/JavaMap.java:14]: > It is recommended that the name of the collections and maps estart with singular name and them the name of the interface, example: numberList

Could you be more specific please :)

File: GabrielaRamirez/GradleExample/workspace/src/main/java/hackkerank/day3/ransomNote/Main.java [hunk 41]
```diff
@@ -10,21 +10,14 @@
...
-//        Result.checkMagazine(magazine, note);
-//
-//        bufferedReader.close();
-//    }
-}}
+        BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(System.in));
+
+        String[] firstMultipleInput = bufferedReader.readLine().replaceAll("\\s+$", "").split(" ");
+
+        int m = Integer.parseInt(firstMultipleInput[0]);
```
Comment (globant-bootcamp) [inline GabrielaRamirez/GradleExample/workspace/src/main/java/hackkerank/day3/ransomNote/Main.java:17]: m? I know you don't have much time, but try not to make the same mistakes that we already reviewed

File: GabrielaRamirez/GradleExample/workspace/src/main/java/hackkerank/day3/ransomNote/Main.java [hunk 42]
```diff
@@ -10,21 +10,14 @@
...
+
+        int n = Integer.parseInt(firstMultipleInput[1]);
```
Comment (globant-bootcamp) [inline GabrielaRamirez/GradleExample/workspace/src/main/java/hackkerank/day3/ransomNote/Main.java:19]: Same

File: GabrielaRamirez/GradleExample/workspace/src/main/java/hackkerank/day1/IfElse.java [hunk 43]
```diff
@@ -1,35 +1,33 @@
...
-        scanner.skip("(\r\n|[\n\r\u2028\u2029\u0085])?");
+        int number = SCANNER.nextInt();
+        SCANNER.skip("(\r\n|[\n\r\u2028\u2029\u0085])?");
 
-        scanner.close();
+        SCANNER.close();
 
-        if (N%2==0 & 2<=N & N<=5){
-            logJava.info("Not Weird");
+        if (number%2==0 & 2<=number & number<=5){
```
Comment (globant-bootcamp) [inline GabrielaRamirez/GradleExample/workspace/src/main/java/hackkerank/day1/IfElse.java:17]: This is the only class that is not in the correct format, remember all binary operators except . should be separated from their operands by spaces. Here you can find more information https://www.oracle.com/java/technologies/javase/codeconventions-whitespace.html

**Review by globant-bootcamp — CHANGES_REQUESTED** [review]

**Review by globant-bootcamp — APPROVED** [review]

**Pull request merged**